dependencies.gradle

```diff
@@ -27,12 +27,12 @@ ext.dep = [
   "mavenPublishGradlePlugin": "com.vanniktech:gradle-maven-publish-plugin:0.8.0",
   "metricsCore": "io.dropwizard.metrics:metrics-core:4.0.2",
   "metricsParent": "io.dropwizard.metrics:metrics-parent:4.0.2",
-  "misk": "com.squareup.misk:misk:0.10.0",
-  "miskHibernate": "com.squareup.misk:misk-hibernate:0.10.0",
-  "miskHibernateTesting": "com.squareup.misk:misk-hibernate-testing:0.10.0",
-  "miskSlack": "com.squareup.misk:misk-slack:0.10.0",
-  "miskTesting": "com.squareup.misk:misk-testing:0.10.0",
-  "miskZipkin": "com.squareup.misk:misk-zipkin:0.10.0",
+  "misk": "com.squareup.misk:misk:0.11.0",
```

Comment:
does this one have the miskweb resources?

Reply:
YES!

Reply:
🙏